ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-ref.yaml

```diff
@@ -10,6 +10,11 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
+  - name: CONTROL_PLANE_REPLICAS
+    default: "3"
+    documentation: |-
+      The number of control plane nodes to create for the cluster.
```

Comment:
Indicate via doc what the expected range of values allowed are.

Reply:
Maybe say "you almost certainly don't want to touch this unless you're on the etcd team" ;)

Reply:
Done